### ISSUE TITLE
XMLType not rendering with Microsoft SQL Server driver

### ISSUE THREAD

**Issue by tedjansen** [TablePlus/TablePlus#2579]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
Microsoft SQL Server 2019

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
Build 396

3. The steps to reproduce this issue:
Connect to a MSSQL server, query a table with XML objects and you get the error:

"Failed to bind column type: 35, please change the protocol on the connection form and try again."

<img width="204" alt="image" src="https://user-images.githubusercontent.com/2352759/147464701-4962ec8a-ece0-472c-a4c3-663ab38503b4.png">

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by huyphams** [comment]
Hi @tedjansen did you "change the protocol on the connection form and try again" as the app suggestion?

**Comment by tedjansen** [comment]
Yes; I tried multiple DB versions; as that is the only thing I can change. It’s DBVersion_74, all versions >74 do not work.

**Comment by Tridie2000** [comment]
@huyphams I'm running into the same issue using all possible protocol versions.

**Comment by mikeburgh** [comment]
Using DB version 70 it worked for an XML column for me against SQL Server 2019

**Comment by Tridie2000** [comment]
Last time I checked I couldn’t find a working protocol version. I’m also limited in my protocol version choices due to this issue: #2755

**Comment by mikeburgh** [comment]
Yes, I just tried 72 and it fails with the same error.. so something in the newer ones breaks XML, but fixes the collation. Has so many nice features in this client, would be good to get this sorted.

**Comment by jwagon** [comment]
Any news on this issue? Having same problem.

**Comment by amstrad** [comment]
Same. Happening now since i upgraded my license  and Tableplus to latest versions. Was fine with my 2021 license version
